### PR TITLE
V1

### PR DESCRIPTION
**Pull request by mxcl**

### CODE REVIEW

**Comment by mxcl** [comment]
an issue here is that `test` is not an ok name for a binary in the `PATH` since the shell built in always claims it.

So we will need another verb or to make them all subcommands again.

**Comment by jhheider** [comment]
> So we will need another verb or to make them all subcommands again.

-x suffix?
`buildx`, `testx`, `cleanx` (lol).

**Comment by mxcl** [comment]
> > So we will need another verb or to make them all subcommands again.
> 
> -x suffix? `buildx`, `testx`, `cleanx` (lol).

gone with `bk build`, `bk test`.

**Comment by jhheider** [comment]
image

File: bin/cmd/docker [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  --tty \
+  --volume "$d:/brewkit" \
+  --volume "$PKGX_PANTRY_PATH:/work" \
+  --volume "${XDG_CACHE_HOME:-$HOME/Library/Caches/pkgx}:/root/.cache/pkgx" \
+  --env PKGX_PANTRY_PATH=/work \
+  --env GITHUB_TOKEN=$GITHUB_TOKEN \
+  --env CLICOLOR_FORCE=1 \
+  --workdir /work \
+  pkgxdev/pkgx \
+  /brewkit/bin/cmd/$CMD "$@"
```
Comment (jhheider) [inline bin/cmd/docker:43]: should we add `--pull` here, to ensure we're up-to-date?

Comment (mxcl) [reply bin/cmd/docker:43]: I think `pkgx` is stable enough now that screw it, but supporting `bk docker --pull build` seems like a nobrainer.

File: share/.DS_Store
Comment (jhheider) [inline share/.DS_Store:1]: should be .gitignored

Comment (jhheider) [reply share/.DS_Store:1]: the one in share/toolchain too. stupid macos stuff.

File: bin/cmd/docker [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+CMD=$1
+shift
+
+mkdir -p "$PKGX_PANTRY_PATH/prefixes/linux"
+
+exec docker run \
+  --name brewkit.pkgx.dev \
+  --rm \
+  --tty $PULL \
```
Comment (jhheider) [inline bin/cmd/docker]: is this variable unused or am i missing it? or is it just snuck in there to make life easy for people who use this a lot?

**Review by jhheider — APPROVED** [review]
looks great. fewer shell scripts means fewer weird shell script bugs.

File: share/.DS_Store
Comment (mxcl) [reply share/.DS_Store:1]: It's in the `.gitignore` now this is actually a deletion.

Comment (jhheider) [reply share/.DS_Store:1]: attention to detail!

File: bin/cmd/docker (hunk 2 above)
Comment (mxcl) [reply bin/cmd/docker]: whoops.

**Pull request merged**